packages/reports/src/Helpers/digits.ts

```diff
@@ -0,0 +1,5 @@
+const digitCategoriesMap: string[] = ['even_odd', 'match_diff', 'over_under'];
+const digitTypesMap: string[] = ['DIGITDIFF', 'DIGITMATCH', 'DIGITOVER', 'DIGITUNDER', 'DIGITEVEN', 'DIGITODD'];
+
+export const isDigitTradeType = (trade_type: string): boolean => digitCategoriesMap.includes(trade_type);
+export const isDigitContractType = (contract_type: string): boolean => digitTypesMap.includes(contract_type);
```

Comment:
```suggestion
const digitCategoriesMap = ['even_odd', 'match_diff', 'over_under'] as const;
const digitTypesMap = ['DIGITDIFF', 'DIGITMATCH', 'DIGITOVER', 'DIGITUNDER', 'DIGITEVEN', 'DIGITODD'] as const;

export const isDigitTradeType = (trade_type: typeof digitCategoriesMap[number]): boolean =>
    digitCategoriesMap.includes(trade_type);

export const isDigitContractType = (contract_type: typeof digitTypesMap[number]): boolean =>
    digitTypesMap.includes(contract_type);

```
@akmal-deriv Although the implementation doesn’t make any sense this way because the function will only accept valid keys so it will always be `true`.
This is one of the cases in which we need to refactor the code instead of adding types to it, We should refactor it and use `.find` instead of `.includes`.

Reply:
Sorry @farzin-deriv, I don't understand what you mean by  `function will only accept valid keys so it will always be true`. The function will accept any string and return whether the input is a digit trade type or if it's a digit contract type. Therefore, there's no need to give any type to input other than string as in that case typescript will prevent using the function in other files unless only correct values are provided. Also, both includes and find would do the same trick, so we can use either of them.